PSMDB-1624-fix-username-and-group-for-different-deplyoments (#1010)
percona/psmdb-docs: docs/kerberos.md

    $ cp /etc/krb5.keytab /etc/mongodb.keytab
    ```

> 3. Change the ownership to the `keytab` file. The user name and group name depend on how PSMDB was installed:

Reviewer: ```suggestion
3. Change the ownership to the `keytab` file. The user name and group name depend on how you installed Percona Server for MongoDB:
```

>     === ":material-linux: Tarball or RPM/DEB packages"

Reviewer: ```suggestion
    === ":material-linux: RPM/DEB packages or tarballs"
```
